opencti-platform/opencti-graphql/src/modules/notification/notification-domain.ts

```diff
@@ -127,15 +134,24 @@ export const triggerDelete = async (context: AuthContext, user: AuthUser, trigge
 };
 export const triggersFind = (context: AuthContext, user: AuthUser, opts: QueryTriggersArgs) => {
   let queryArgs = { ...opts };
-  if (opts.filters?.some((f) => f.key.includes(TriggerFilter.UserIds))) {
+  // key is a string[] because of the resolver, we have updated the keys
+  const userIdFilter = opts.filters?.find((f) => (f.key as string[]).includes('authorized_members.id'));
```

Comment:
I don't understand this line
We don't have authorized_members as filter key in GraphQL

Reply:
`authorized_members` is the filter in elastic, but it is not a is not functionally comprehensible. So in the api we use `group_ids` and `user_ids` (resolver, line 38) :
```
  TriggerFilter: {
    user_ids: 'authorized_members.id',
    group_ids: 'authorized_members.id',
  },
```